scipy/stats/_multivariate.py

```diff
@@ -3775,6 +3813,51 @@ def rvs(self, eigs, random_state=None, tol=1e-13, diag_tol=1e-7):
 random_correlation = random_correlation_gen()
 
```

Comment:
While we're here, can you add documentation of the parameter `random_state` to the `rvs` method above? Also in the top level documentation of the class, add `tol` and `diag_tol` as parameters of the `rvs` method. Not strictly in scope here, but it would be good to do this while we're working here.

Reply:
Never mind. There is a _lot_ of documentation work needed; we'll take care of it as part of that effort.